tests/api/test_responses.py

```diff
@@ -139,7 +139,12 @@ def test_pass_through(simple_app):
     app_client = simple_app.app.test_client()
 
     response = app_client.get("/v1.0/multimime", data={})  # type: flask.Response
-    assert response.status_code == 200
+    assert response.status_code == 500
+    assert (
```

Comment:
This is a change of behavior. If the spec defines multiple content types, we should not guess.